benchmark/camelyon/requirements.txt

```diff
@@ -1,6 +1,6 @@
 tqdm==4.66.3
-numpy==1.24.3
-scikit-learn==1.5.0
-torch==2.0.1
-torchvision==0.15.2
+numpy==1.26.4
+scikit-learn==1.5.1
+torch==2.3.0
+torchvision==0.18.0
 --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
Is this set of requirements compatible with python versions <3.12?

Reply:
yes, it's matching what is already used in the documentation examples